client/src/constants/lab11/FogIndexCalculationData.js

```diff
@@ -0,0 +1,36 @@
+const FogIndexCalculationData = {
+  words: [
+    {
+      id: "0",
+      key: "wordsInput0",
+      variableName: "wordCount",
+      comment: `// Enter 'letterContent.split(' ').length' for the correct implementation of the word count.`,
```

Comment:
Splitting on '.' adds one more empty array after the last period, need to add a -1

Reply:
@Mster115